.circleci/config.yml

```diff
@@ -87,7 +87,7 @@ references:
       - <<: *doctl_authenticate
       - deploy:
           name: Install Codacy
-          command: make -C .doks/ deploy_to_doks
+          command: make -C .doks/ deploy_to_doks RELEASE_NAME=$RELEASE_NAME NAMESPACE=$NAMESPACE
```

Comment:
I don't think you need to pass the env variables

Reply:
Fixed